linux_os/guide/services/ssh/ssh_server/sshd_use_approved_macs_ordered_stig/oval/ubuntu.xml

```diff
@@ -0,0 +1,78 @@
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("Limit the Message Authentication Codes (MACs) to those which are FIPS-approved.") }}}
+   <criteria comment="sshd is configured correctly or is not installed" operator="OR">
```

Comment:
the "approved" rules should have a requirement on FIPS. See the shared OVAL.

Reply:
Nice catch, thanks. I added the requirement and rebased to master.
The tests currently fail on Ubuntu 22.04 since the OS is not yet listed in `installed_OS_is_FIPS_certified`.
If added to the list, all tests pass.